### ISSUE TITLE
`gen` command is a bit confusing, as there is a `gen` package on npm as well

### ISSUE THREAD

**Issue by Robbie-Cook** [CloudCannon/bookshop#11]
https://www.npmjs.com/package/gen

**Comment by Robbie-Cook** [comment]
Proposed fix: convert 
``` 
"bin": {
    "gen": "src/index.js"
 }
```

to:

``` 
"bin": {
    "bookshop-gen": "src/index.js"
 }
```

**Comment by bglw** [comment]
Addressed at #20 if you want to review that

**Comment by Robbie-Cook** [comment]
Great, will close on merge :)
